app/src/crypto.c

```diff
@@ -28,7 +29,13 @@ uint8_t hdPathLen;
 
 #include "cx.h"
 
-zxerr_t  crypto_extractPublicKey(const uint32_t path[HDPATH_LEN_DEFAULT], uint8_t *pubKey, uint16_t pubKeyLen) {
+void keccak(uint8_t *out, size_t out_len, uint8_t *in, size_t in_len){
```

Comment:
if this is a private function, you can add static and also checking NULL pointer and sized would be a good improvement